src/components/InviteFriends.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { get } from 'lodash/object'
+import TextField from '@material-ui/core/TextField'
+
+const InviteFriends = (props) => {
+  const getReferralUrl = () => {
+    const { baseURL, user } = props
+    const username = get(user, 'username')
+    const referralUrl = username
+      ? `${baseURL}/?u=${encodeURIComponent(username)}`
+      : baseURL
+    return referralUrl
+  }
+
+  const { user, helperText, label } = props
+  const referralUrl = getReferralUrl()
+  const textFieldRef = React.createRef()
+
+  const highlightReferralUrl = () => {
+    textFieldRef.current.select()
+  }
+
+  const getHelperText = () => {
+    if (helperText !== '') {
+      return helperText
+    }
+
+    return get(user, 'username')
+      ? `and you'll get 350 Hearts for every person who joins!`
```

Comment:
this copy is out of date.  Not sure what it should be tho

Reply:
Sounds reasonable. @kmjennison thoughts?

Reply:
@jedtan We actually don't need any text/label at the moment, so remove the default text altogether.